e2e/operation-field-permissions/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@e2e/operation-field-permissions",
+  "private": true,
+  "dependencies": {
+    "@envelop/core": "^5.0.2",
```

Comment:
Do we always need that?

Reply:
We need it because:
1. the example converter copies the package.json, so we want to include all peers there for the example to work as expected
1. core is a peer dep of the plugin - without it in the package.json, yarn will raise warnings